README.md

```diff
@@ -1,120 +1,67 @@
+A context graph is a point-in-time snapshot of the cluster’s state. Clients of the Cluster Insight service, such as user interfaces, can retrieve context graphs through the service's REST API. Each call may produce a different context graph, reflecting the inherent dynamicity in the Kubernetes cluster.
 
 A context graph provides contextual information that can be combined with resource level monitoring data to enable visual navigation of the dynamic state of a Kubernetes cluster.
 
-The nodes of the context graph are cluster resources (e.g. nodes, pods, services, replication-controllers, containers, processes, and images), and the edges are the inferred relationships (e.g. contains, runs, monitors, loadBalances, createdFrom).
+The nodes of the context graph are cluster resources (cluster, nodes, services, replication controllers, pods, containers, and images), and the edges are the inferred relationships (contains, runs, monitors, loadBalances, createdFrom).
 
-## How to install and access the service
+## How to set up and access the service
 
-The cluster-insight service uses a Docker container, available [here](https://registry.hub.docker.com/u/kubernetes/cluster-insight/) on Docker Hub.
-The container runs in one of two modes, specified by an environment variable:
+Deploying Cluster Insight involves creating a service and a replication controller. The replication controller creates one pod, which contains one container. The container image is [here](https://registry.hub.docker.com/u/kubernetes/cluster-insight/) on Docker Hub.
 
-* In "master" mode, it collects data from Kubernetes and from the Docker daemons running on the cluster nodes through proxies. 
-* In "minion" mode, it acts as a proxy for the local Docker daemon.
+### Setup
 
-An instance of the container in "master" mode should run on at least one node in the cluster, and an instance in "minion" mode should run on every node.
+Assuming you have configured `kubectl` to talk to your Kubernetes cluster, deploying Cluster Insight is a simple matter of invoking `kubectl create`:
 
-The cluster-insight service does not require any changes to the cluster, and should run correctly on any valid  cluster without modification. Note, however, that it has only been tested on GCE clusters running Debian GNU/Linux 7 (wheezy) and vagrant clusters running Ubuntu 14.04.
-
-The provided installation script will configure, install, and run cluster-insight. You should be able to run it on any Linux or MacOSX machine. Note, however, that it has only been tested on Ubuntu 14.04 and MacOSX Yosemite.
-
-### Kubernetes API version
-
-The cluster-insight service currently supports only Kubernetes API version "v1".
-To check which API versions are supported by your cluster, run the following command on the master node of your cluster:
-```
-$ kubectl api-versions
 ```
```

Comment:
Adding a comment that shows the user how to have the correct context for the project/cluster would be useful. For e.g.

`$ kubectl config use-context gke_<project-id>_<zone-name>_<cluster-name>`